documentation/API-authentication-token.md

```diff
@@ -29,12 +31,10 @@ And if the token request was successful, the token response could be:
 "access_token":"[ACCESS_TOKEN]",
 "token_type":"Bearer",
 "expires_in":86400,
+"schoolidentifier": "digideliveryid"
```

Comment:
Do we need this in the reply ?

Reply:
To me it reads as that it is optional (since _the token response could be_) , and for Noordhoff we will first focus on getting the info in the access_token and other changes. I can imagine that we will not implement this.

Reply:
If it would be optional I would prefer to remove it from the response body (in this specification).